### ISSUE TITLE
Problems with using Keras' validation loss 

### ISSUE THREAD

**Issue by jhung0** [broadinstitute/keras-rcnn#196]
Using Keras' validation loss (i.e. setting validation_data parameter in model.fit_generator) could lead to an Unimplemented error, so don't use it
